### DISCUSSION TITLE
tar container is corrupt, connection stalled or similar

### DISCUSSION THREAD

**Discussion by Antguz** [davidfrantz/force#110] [Q&A] [answered]
Hello,

I am processing several compressed (.tar) Landsat files downloaded from earthexplorer.usgs.gov using the bulk download application. The processing of these goes very smoothly, but I started to noticed that many of the log files (~40% of 2160) say 'tar container is corrupt, connection stalled or similar'.

I had uncompressed some of these files to see if they are corrupted, and they are not. I can process them using QGIS for instance. That mostly happened on Landsat 7 and 8, but not on 4 or 5.

Not sure what is going on. I can share a couple of the files if you wish.

What could be a potential solution? uncompress all of them myself a do a queue using the folder path.

Thanks in advance.

**Comment by ernstste** [comment]
Hi,

I'm not sure what the cause of your issue is, maybe David has an idea.
Assuming that the archives are not damaged, your proposed solution will work as a workaround.

Stefan

**Comment by davidfrantz** [comment, marked as answer]
Hi @Antguz 

can you please post how many CPUs you have available, and upload your parameter file?

Cheers,
David

**Reply by Antguz** [reply]
Hi @davidfrantz,

I think that I solved the problem. I am running this using a 3950x AMD which has 32 virtual CPUs (16 x 2) with 128 GB of RAM. However, I had storage the Landsat images in a SATA 6.0Gb/s 3.5 disk attached to my PC. The errors seem to come from the speed of the hard drive (uncompress, reading and writing files). I suspect that seems to happen more frequently on Landsat 7 and 8 since the files are bigger. I solved this by passing my files to my SSD drive. This is not a bug at all.

Thank you for being willing, I hope this discussion helps someone who is going through the same thing. 

Regards.
Antonio
